pydoclint/utils/arg.py

```diff
@@ -431,3 +431,12 @@ def hasTypeHintInAllArgs(self) -> bool:
         they don't need to have type hints.
         """
         return all(_.hasTypeHint() for _ in self.infoList if _.notStarArg())
+
+    def insertAt(
+            self,
+            index: int,
+            arg: Arg,
+    ) -> None:
+        """Insert an Arg at a specific index."""
+        self.infoList.insert(index, arg)
+        self.lookup[arg.name] = arg.typeHint
```

Comment:
Are there any safeguards when `arg` already exists in `self.infoList` and `self.lookup`?